packages/patrol_cli/lib/src/pubspec_reader.dart

```diff
@@ -19,9 +20,10 @@ class PatrolPubspecConfig with EquatableMixin {
   AndroidPubspecConfig android;
   IOSPubspecConfig ios;
   MacOSPubspecConfig macos;
+  String testFileSuffix;
 
   @override
-  List<Object?> get props => [android, ios, macos];
+  List<Object?> get props => [android, ios, macos, testFileSuffix];
```

Comment:
In the future we'll likely want to add another class for common (i.e. Flutter-side-only) configuration, but for now this `testFileSuffix` is good enough.